### PR TITLE
Doc: improving readme and contributing guide

### PR DESCRIPTION
**Pull request by RMCampos**
# Description

Improve readme and contributing guide.

## Type of change

- [x] Documentation

# How Has This Been Tested?

No tests are needed.

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have already been accepted and merged

## Further comments

No additional comments.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -44,22 +45,39 @@ We recommend IntelliJ IDEA Community, because all of its plugins and
 configurations possibilities, here's [the website](https://www.jetbrains.com/idea/download).
 But feel free to use Eclipse or other IDE of your choice.
 
-## Follow our best practices
+### Check-style
```
Comment (mgaseta) [inline CONTRIBUTING.md:48]: There is an extra '#' here, it can be removed to remain in the same pattern as the others.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -44,22 +45,39 @@ We recommend IntelliJ IDEA Community, because all of its plugins and
...
+After setting up your environment you might want to see this service running. 
+You can get it up and running by typing `./mvnw spring-boot:run` in the project
+root directory.
+
+Note: TODO add here about database and running HOW-TOs.
 
 ## Run tests
 
-You can run tests running `make tests`.
+You can run tests running `./mvnw test`. Tests coverage reposts can be seen
```
Comment (mgaseta) [inline CONTRIBUTING.md:65]: Small typo, 'reposts' -> 'reports'.

**Review by mgaseta — CHANGES_REQUESTED** [review]
Just 2 small typos, besides that, it looks great!

File: CONTRIBUTING.md (hunk 1 above)
Comment (RMCampos) [reply CONTRIBUTING.md:48]: Actually, is correct. The "Check-style" item is nested with "Set up your environment". So three is the right one.

File: CONTRIBUTING.md (hunk 2 above)
Comment (RMCampos) [reply CONTRIBUTING.md:65]: Thanks. Fixed.

**Review by mgaseta — APPROVED** [review]

**Pull request merged**